.travis.yml

```diff
@@ -2,4 +2,4 @@ language: node_js
 node_js: 6.9.1 # latest LTS
 cache:
   directories:
-    - phantomjs
+    - node_modules/phantomjs-prebuilt
```

Comment:
When cache is restored, this will create a `node_modules` directory. Then, `make build` is a no-op and will never run `npm install`.